twill-yarn/src/main/java/org/apache/twill/internal/appmaster/RunnableProcessLauncher.java

```diff
@@ -76,6 +76,11 @@ public String toString() {
 
     return new ProcessController<R>() {
       @Override
+      public void close() throws Exception {
+        // no-op
```

Comment:
I just meant, now both close and cancel will kill the runnable?

On Aug 26, 2016 12:55 AM, "Terence Yim" <notifications@github.com> wrote:

> In twill-yarn/src/main/java/org/apache/twill/internal/appmaster/
> RunnableProcessLauncher.java
> <https://github.com/apache/twill/pull/6#discussion_r76378304>:
>
> > @@ -76,6 +76,11 @@ public String toString() {
> >
> >      return new ProcessController<R>() {
> >        @Override
> > +      public void close() throws Exception {
> > +        // no-op
>
> Controller ties to the lifecycle of the stuff that it is controlling. What
> do mean by leave the runnable process running when closing the controller?
> Is it related to #4 <https://github.com/apache/twill/pull/4> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/twill/pull/6/files/7b7f472f52e17d5742c88f6818ba4150a05003b5..f5716b0e5765604427225b1716d75ea590c7aaee#r76378304>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACU_EUN0tSXqJYX-qeGQOWRmzPOpQXYCks5qjpv6gaJpZM4JtyZN>
> .
>